datumaro/datumaro/plugins/accuracy_checker_plugin/details/representation.py

```diff
@@ -0,0 +1,62 @@
+
+# Copyright (C) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+from datumaro.util.tf_util import import_tf
+import_tf() # prevent TF loading and potential interpeter crash
+
+import accuracy_checker.representation as ac
+
+import datumaro.components.extractor as dm
+from datumaro.util.annotation_tools import softmax
+
+def import_predictions(predictions):
+    # Convert Accuracy checker predictions to Datumaro annotations
+
+    anns = []
+
+    for pred in predictions:
+        anns.extend(import_prediction(pred))
+
+    return anns
+
+def import_prediction(pred):
+    if isinstance(pred, ac.ClassificationPrediction):
+        scores = softmax(pred.scores)
+        return (dm.Label(label_id, attributes={'score': float(score)})
+            for label_id, score in enumerate(scores))
+    elif isinstance(pred, ac.ArgMaxClassificationPrediction):
+        return (dm.Label(int(pred.label)), )
+    elif isinstance(pred, ac.CharacterRecognitionPrediction):
+        return (dm.Label(int(pred.label)), )
+    elif isinstance(pred, (ac.DetectionPrediction, ac.ActionDetectionPrediction)):
+        return (dm.Bbox(x0, y0, x1 - x0, y1 - y0, int(label_id),
+                attributes={'score': float(score)})
+            for label, score, x0, y0, x1, y1 in zip(pred.labels, pred.scores,
+                pred.x_mins, pred.y_mins, pred.x_maxs, pred.y_maxs)
+        )
+    elif isinstance(pred, ac.DepthEstimationPrediction):
```

Comment:
not sure that it is good approach, at least it will require update on each new representation and extend code with a lot of elif statements...
Probably, condition can be converted to dict format where keys are type and value is functor for conversion or something similar.
Another approach which I can suggest to have to_dm_prediction method inside AC prediction, if it is possible (but it will require to update AC code and make AC dependent on DM...now we are working on dep reducion due to necessary to work on ARM...)

so here is something to think about...